src/main/com/example3/radek/ArrayRandomNumberGeneratorApplication.java

```diff
@@ -0,0 +1,39 @@
+package main.com.example3.radek;
```

Comment:
melo by to jit nove do package example003

Reply:
Hotovo